daffodil-runtime1-unparser/src/main/scala/org/apache/daffodil/processors/unparsers/SequenceChildUnparsers.scala

```diff
@@ -91,8 +91,6 @@ abstract class RepeatingChildUnparser(
     // But an expression might still refer to dfdl:occursIndex() and we want that
     // to be 1, not wherever an enclosing parent array is.
     //
```

Comment:
Should this comment be removed now? Since that's now handled elsewhere?

Reply:
Yes. Comment is misleading now. Removing.